pnpm-lock.yaml

```diff
@@ -643,6 +643,7 @@ importers:
       prop-types: 15.8.1
       react: 17.0.2
       react-dom: 17.0.2
+      refx: 3.1.1
```

Comment:
We could remove `refx` since we're using `thunk` now.

Reply:
addressed here https://github.com/Automattic/jetpack/pull/22448/commits/a1d2b9f1d2cd8e77d85ff27a43c1ea3e8921a47d